pkg/k8s/deployments/translate.go

```diff
@@ -428,7 +428,7 @@ func TranslateOktetoInitBinContainer(spec *apiv1.PodSpec) {
 	c := apiv1.Container{
 		Name:            oktetoBinName,
 		Image:           oktetoBinImageTag,
-		ImagePullPolicy: apiv1.PullIfNotPresent,
+		ImagePullPolicy: apiv1.PullAlways,
```

Comment:
this is for the bin image right? if so, could we leave it as it was so it reduces the time for the `up`?